app/lib/team.js

```diff
@@ -133,12 +136,14 @@ async function listModeratedBy (osmId) {
 * @param {string} data.name - name of the team
 * @param {geojson} data.location - lat/lon of team
 * @param {int} osmId - id of first moderator
+* @param {object} conn - optional parameter for database connection to re-use connection in case of nested
```

Comment:
Consider renaming  this param from `conn` to `trx`. Mainly for clarity because  `conn` and `trx` actually have the same interface.

Also in JsDoc, consider adding JsDoc [optional type tags](https://jsdoc.app/tags-type.html) 

```
An optional parameter named foo.
@param {number} [foo]
// or:
@param {number=} foo
An optional parameter foo with default value 1.
@param {number} [foo=1]
```

This helps a lot in the IDE where it uses the JsDoc for type hints and parameter validation.